ULC/Repl.hs

```diff
@@ -45,10 +46,12 @@ settings :: Settings (StateT Environment IO)
 settings = Settings contextCompletion Nothing True
   where
     contextCompletion :: CompletionFunc (StateT Environment IO)
```

Comment:
can you add a quick comment for each of these please, it will help those who see this in the future